static/chats.js

```diff
@@ -144,6 +144,12 @@ function saprotiKomandu(teksts) {
     case "/v":
       zinja = "Javascript versija: " + VERSIJA;
       break;
+    case "/es":
```

Comment:
Šis neko nedod, jo nav ielikts:
```js
break
```
Attiecīgi mēs ieejam if/else un aizejam tālāk un mums atgriež "/help", jo tas ir pēdējais, kur ir break. Tad mēs izejam un switch ārā un beigās ar return atgriežam pēdējo.